src/django/api/migrations/0161_create_disable_list_uploading_switch.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.17 on 2024-11-13 13:59
```

Comment:
Pls, regenerate migration instead of renaming it in sake of consistency. This will re-generate this description as well.

Reply:
done